src/components/tab-title/tab-title.tsx

```diff
@@ -59,6 +74,12 @@ export class TabTitle implements InteractiveComponent {
     }
   }
 
+  /** When `true`, a close button is added to the component. */
+  @Prop({ reflect: true }) closable = false;
+
+  /** When `true`, does not display or position the component. */
```

Comment:
+@geospatialem for editorial review.

Reply:
Is the behavior that the component won't be displayed? If so, perhaps similar to [List item](https://github.com/Esri/calcite-components/blob/master/src/components/list-item/list-item.tsx#L81-L82)?:

> When `true`, hides the component.